test/run-drun/variant-upgrades/version1.mo

```diff
@@ -0,0 +1,23 @@
+import Prim "mo:prim";
+
+// Compatible upgrade
+actor {
+  type Variant = {
+    #two : { key : Nat; var name : Text };
```

Comment:
Maybe worth adding incompatible version3.mo, with same variants as version1.mo, but an additional field for variant #two, checking deep incompatibility of the variant type.

Reply:
Thanks for the suggestion. I refined the test accordingly.